pkg/plugins/helm/provider.go

```diff
@@ -461,6 +477,30 @@ func (p *Provider) getKubePodsAsClusterNodes(runtimeId, namespace, clusterId, cl
 	return pbClusterNodes, nil
 }
 
+func (p *Provider) updateClusterEnv(job *models.Job) error {
```

Comment:
The env has already been updated in cluster handler

Reply:
I think we should updated env after the job is finished successfully.

Reply:
Ok, then I should remove the update db from cluster handler?

Reply:
I will fix this.

Reply:
OK